test/time_unittest.cc

```diff
@@ -156,4 +156,26 @@ TEST(TimeTest, FromStringScientificSeconds) {
   );
 }
 
+TEST(TimeTest, SecondsSinceEpoch) {
+  const time_point t =
+      time::rfc3339::FromString("2017-07-14T02:40:00Z");
+  EXPECT_EQ(1500000000, time::SecondsSinceEpoch(t));
+}
+
+TEST(TimeTest, SafeLocaltime) {
```

Comment:
Is the only purpose of safe local time to be thread safe? If so, should we test it by firing off multiple threads and ensure they all also receive the same time?

Reply:
Yeah, I'm actually not sure we need to test these functions at all given that they are trivial wrappers.

Reply:
The wrapper methods do seem trivial. But +1 on what Bryan points out, if we want to test these, the thread safety should be the target.

Reply:
Any multi-threaded tests will be fragile and flaky (i.e., it's very hard to make something like this fail). The only thing we can do is test that `SafeLocalTime` returns results identical to `std::localtime` (which is its contract).

Reply:
Acknowledged.